sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/TypeScriptRules.java

```diff
@@ -181,8 +180,8 @@
   private final Map<String, RuleKey> tsLintKeyToRuleKey;
   private final Map<String, TypeScriptRule> tsLintKeyToRuleObject;
 
-  public static void addToRepository(NewRepository repository, SonarRuntime runtime) {
-    RuleMetadataLoader ruleMetadataLoader = new RuleMetadataLoader(RESOURCE_FOLDER, DEFAULT_PROFILE_PATH, runtime);
+  public static void addToRepository(NewRepository repository) {
+    RuleMetadataLoader ruleMetadataLoader = new RuleMetadataLoader(RESOURCE_FOLDER, DEFAULT_PROFILE_PATH);
```

Comment:
this way we will not properly process security hotspots. Do we have them?
Or use latest build of commons

Reply:
I use the latest build, we should release it though

Reply:
I will take care of it today